moveit_ros/visualization/rviz_plugin_render_tools/src/trajectory_visualization.cpp

```diff
@@ -34,6 +34,9 @@
 
 /* Author: Dave Coleman */
 
+#include <boost/algorithm/string.hpp>
+#include <boost/algorithm/string/replace.hpp>
+
```

Comment:
Are these new includes needed? There seems to be no change to in the source that matches.